core/ide/src/main/java/org/alice/stageide/sceneeditor/interact/croquet/AbstractSetLocalTransformationActionOperation.java

```diff
@@ -77,6 +77,7 @@ public AbstractSetLocalTransformationActionOperation(Group group, UUID individua
 
   protected void setLocalTransformation(AffineMatrix4x4 lt) {
     AbstractTransformable sgTransformable = getSGTransformable();
+    if (sgTransformable == null) return;
```

Comment:
A tiny thing, but please add brackets around the return since this violates the style check and prevents the build.